### ISSUE TITLE
Kryo generates warning in JDK-11

### ISSUE THREAD

**Issue by dturanski** [spring-projects/spring-statemachine#862]
see https://github.com/spring-cloud/spring-cloud-skipper/issues/971

**Comment by jvalkeal** [comment]
Not really sure what we should do all these warnings. We get some with a gradle build stuff and there's warning from eclipse uml libs as well.

**Comment by dturanski** [comment]
There's a system property to set for Kryo to ignore the warnings, but I'm not sure which component would do that.  We get similar warnings from Spring reflection as well. So...
________________________________

**Comment by jvalkeal** [comment]
Yeah, we're not going to a business of setting or forcing system props. Hopefully there's some fix in kryo at some point.

**Comment by Daanielvb** [comment]
I believe that SSM would be greatly benefited from upgrading kryo to version 5.x @jvalkeal. Are there any plans for it. Here is what they're saying at kryos project[https://github.com/EsotericSoftware/kryo/pull/727#issuecomment-900907744](url)
